Assets/Scripts/MainMenu.cs

```diff
@@ -5,20 +5,36 @@
 
 public class MainMenu : MonoBehaviour
 {
+    int current = 1;
+    int total = 0;
+    
     // Start is called before the first frame update
     public void PlayGame(){
 
-        SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex +1);
+        SceneManager.LoadScene(SceneManager.GetActiveScene().buildIndex + current);
     }
 
-    public void QuitGame(){
+    // public Update(){
```

Comment:
Can we please delete this method since it's commented out the // public Update()?